### PR TITLE
feat landing page

### PR DESCRIPTION
**Pull request by SuyashTiwariCoder**
add landing page

### CODE REVIEW

**Comment by SuyashTiwariCoder** [comment]
LGTM

File: index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      
+          <h1>SUPER UI</h1>
+        <ul class="navigation__list">
+          <li ><a class="navigation__list--item "href="#">Home</a></li>
+          <li ><a class="navigation__list--item" href="#">Docs</a></li>
+          <li><a class="navigation__list--item" href="#">Dark</a></li>
+        </ul>
+      </nav>
+  <div class="container">
+    <img src="/Images/undraw_building_blocks_re_rwls.svg" alt="">
```
Comment (dev-enforced) [inline index.html]: In images it is good to add alt atributes for better accessibility

**Review by dev-enforced — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (SuyashTiwariCoder) [reply index.html]: done

**Review by dev-enforced — APPROVED** [review]
LGTM!!

**Pull request merged**